modules/weko-items-ui/weko_items_ui/static/js/weko_items_ui/app.js

```diff
@@ -539,6 +559,160 @@ function toObject(arr) {
         return fileMetaForm;
       };
 
+      $scope.searchKey = function(item) {
+        for (let key in $rootScope.recordsVM.invenioRecordsSchema.properties) {
+          var value = $rootScope.recordsVM.invenioRecordsSchema.properties[key];
+          var properties = value.properties ? value.properties : (value.items ? value.items.properties : [])
+          if (Object.keys(properties).indexOf(item) >= 0) {
+              if ($scope.authors_keys.indexOf(key) >= 0 ) {
+                break
+              }
+             $scope.authors_keys.push(key);
+          }
+        }
+      };
+      $.ajax({
+        url: '/api/items/author_prefix_settings',
+        method: 'GET',
+        async: false,
+        success: function (data, status) {
+          $scope.data_author = data;
+        },
+        error: function (data, status) {
+        }
+      });
```

Comment:
This source code always calls API when open Item Registration Page.
Move ajax in a function and call it when you want to call API.